src/Request/WebHelper.php

```diff
@@ -97,11 +97,11 @@ public function put($endpoint, $queryParams, array $options = array(), array $da
          * Send a PUT requst using cURL
          * @param string $url to request
          * @param array $queryParams values to send
-         * @param array $options for cURL
          * @param string $filepath of image to add
+         * @param array $options for cURL
          * @return string
          */
-        public function putImageRequest($endpoint, $queryParams, array $options = array(), string $filepath) {
+        public function putImageRequest($endpoint, $queryParams, string $filepath, array $options = array()) {
```

Comment:
An alternative would be to make `string $filepath = ''` optional, too.

Reply:
I think `$filepath` is required.